Prerelease 0.3.0

hyperthunk opened the pull request:
Release candidate - with all the recent fixes included, I'd like to publish to hackage so that we can depend on the latest version of the code in -async and -client-server, etc. I believe the breaking API changes require a minor version bump?

@qnikst et al, please shout if you disagree that this should become 0.3.0, or think we should include other fixes or changes...

hyperthunk committed:
tweak tested GHC versions

hyperthunk committed:
change maintainer

hyperthunk committed:
bump version number

qnikst commented:
I don't see API breaking changes that I'd like to happen before release, so LGTM.

facundominguez committed:
Update maintainer field.

agentm committed:
adjustments to compile with GHC 8.0.1

unable to run tests due to missing systest dependency

qnikst committed:
Merge pull request #15 from agentm/master

adjustments to compile with GHC 8.0.1

qnikst committed:
Drop use of GHC<7.6.

Commit:
Fixed number of hours in a day

Signed-off-by: Blake Rain <blake.rain@inchora.com>

qnikst committed:
Merge pull request #16 from BlakeRain/master

Fixed number of hours in a day

Commit:
Adds stack.yml and relaxes some constraints to enable compilation using lts-7.14.

qnikst committed:
Merge pull request #17 from diogob/add_stack

Adds stack.yml and relaxes some constraints to enable compilation using lts-7.14.

3noch committed:
Bump version bounds on time

qnikst committed:
Merge pull request #18 from 3noch/master

Bump version bounds on time

hyperthunk committed:
Fix travis build (#20)

- Copy the distributed-process travis config, but drop cabal support
- loosen build constraints, fix missing flags
- stop supporting 7.8.4 in CI

hyperthunk committed:
always build against the latest nightly
this way we're not waiting for an issue to be raised by the stackage team
should something go wrong

hyperthunk committed:
fix(travis-ci): stack config for 8.0.1, travis updates

hyperthunk committed:
upload to hackage when releases are tagged

hyperthunk committed:
fix(haddock): remove leading whitespace

hyperthunk committed:
tidy

hyperthunk committed:
fix(travis-ci): helps if you get the shell syntax right...

hyperthunk committed:
preparatory commit for release 0.3.0

hyperthunk committed:
fix merge conflicts

hyperthunk committed:
Update README

hyperthunk committed:
Update README
